src/runtime/wasm.ts

```diff
@@ -457,6 +464,11 @@ export class WasmContainer {
     this.particleMap.set(innerParticle, particle);
   }
 
+  // Allocates memory in the wasm container; the calling particle is responsible for freeing.
+  resolve(urlPtr: WasmAddress) {
```

Comment:
Let's add the return type: `: WasmAddress`

Reply:
Done.